db/migrate/20220717185757_add_original_creators_to_work.rb

```diff
@@ -0,0 +1,51 @@
+class AddOriginalCreatorsToWork < ActiveRecord::Migration[6.0]
+  def up
+    if Rails.env.staging? || Rails.env.production?
+      database = User.connection.current_database
+
+      puts <<~PTOSC
+        Schema Change Command:
+
+        pt-online-schema-change D=#{database},t=works \\
```

Comment:
@redsummernight @tickinginstant how comfortable are you folks with
1. Adding new fields to this table in general; and
2. Scanning through in the background cleanup job?
For the second, an index might be helpful although maybe not necessary since it is, after all, a background job. Another option I've been mulling over is to add a new table with this information separately; that could also be a slightly nicer way to store multiple orphaners. Thoughts?

Reply:
I'm personally kind of iffy about adding more columns to the works table in this case, and would prefer it to be stored in a different table with one row per former creator (which is what I think you're suggesting as an alternative). As you say, it handles multiple orphanings in a cleaner way, because you can have a separate date associated with each former creator.

Reply:
Ok, I've updated here with a somewhat basic implementation using a new table. I know it was mentioned earlier that we only need this for `Work`, but since this method would be harder to extend to other `Createable`s I've done some exploration of a potential polymorphic approach in https://github.com/brianjaustin/otwarchive/tree/orphan-alt-polymorphic. What do you think?

Reply:
I can't speak for PAC, so you should probably check with someone who can, but I suspect that it won't need to handle chapters or series -- you can't orphan just a chapter, and orphaning a series also orphans the works in it, so in 99.9% of cases all the critical information will be available on the works.

Also, I think aside from table names/column names, it's not too hard to transition to polymorphism if it ends up being needed for more than works. (Just add a new non-null column with a default value `Work`, and run the migration before deploy. The table's probably going to pretty small, so we probably wouldn't even need to do ptosc for the migration.)

Reply:
Fair enough! From @redsummernight's previous comment
> We only care about this info on works for now, so you don't have to generalize the approach in Creatable yet.

so I'm going to resolve this thread for now.